susemanager-utils/susemanager-sls/src/tests/test_beacon_pkgset.py

```diff
@@ -22,12 +22,12 @@ def test_validate():
     '''
     Test validate() function
     '''
-    res, msg = pkgset.validate({'cookie': '/bogus/path'})
-    assert res is True
-    assert msg == 'Configuration validated'
-
-    for cfg in [{}, {'bogus': 'data'}]:
-        res, msg = pkgset.validate(cfg)
+    with patch.object(pkgset.os.path, "exists", MagicMock(return_value=True)):
```

Comment:
nitpick, no need to change: I think a regular `Mock` would be enough here as `os.path.exists` is a function and not a class with dunder methods.